### PR TITLE
Routing orgs and bases

### PR DESCRIPTION
**Pull request by mcgnly**
In this branch, I set up routing for specific orgs and bases, which we will get eventually from the backend. As such, they are currently hardcoded. It also brings in tailwind as a css utility for easier styling, let's see how that goes and if we like it!

### CODE REVIEW

**Comment by mcgnly** [comment]
[Create routes for the organisation and bases from the data in the `organisations` db-table and the `camps` db-table. #python #flask #react](https://trello.com/c/cuqHeRbF/464-create-routes-for-the-organisation-and-bases-from-the-data-in-the-organisations-db-table-and-the-camps-db-table-python-flask-rea)

**Review by DurkoMatko — APPROVED** [review]

**Review by jamescrowley — APPROVED** [review]

File: tailwind.config.js [hunk 1]
```diff
@@ -0,0 +1,706 @@
+module.exports = {
```
Comment (jamescrowley) [inline tailwind.config.js:1]: This seems very large. Have you customised all of this, or is this the defaults? My tailwind.config.js is much smaller.... 😁

Comment (mcgnly) [reply tailwind.config.js:1]: nope! This is all the defaults... I can cut it down though. This is the first project I'm using tailwind on :)

Comment (mcgnly) [reply tailwind.config.js:1]: adding a purge item to the tailwind config based on this: 
https://tailwindcss.com/docs/controlling-file-size/

**Pull request merged**